tests/unit/EventsAndDelegatesTest/AnonymousFunctionImplementationTests/AnonymousFunctionImplementationTests.cs

```diff
@@ -0,0 +1,32 @@
+namespace AnonymousFunctionImplementationTests
+{
+    using EventsAndDelegates;
+    public class AnonymousFunctionImplementationTests
+    {
+        AnonymousMethodsImplementation? anonymousMethodsImplementationObject;
+
```

Comment:
```suggestion
public AnonymousFunctionImplementationTests()
{
    this.anonymousMethodsImplementationObject = new();
}
```

So that we dont need to declare the instance everytime

Reply:
Used the following practice in needed tests